heron/tools/ui/resources/static/js/physical-plan.js

```diff
@@ -63,16 +63,23 @@
       container.children = _.sortBy(container.children, 'name');
       // Parse index
       container.index = parseInt(container.id.split('-')[1]);
-      // Search for resource config in packing plan
+      // Search for container resource config in packing plan
       container.required_resources = {
-        'cpu': 0,
-        'disk': 0,
-        'ram': 0
+        'cpu': 'unknown',
+        'disk': 'unknown',
+        'ram': 'unknown'
       };
       for (var i in containerPlan) {
         var packing = containerPlan[i];
         if (packing.id === container.index) {
           container.required_resources = packing.required_resources;
+          // Get instance resources
+          var instance_resources = {};
+          for (var j in packing.instances) {
+            var inst = packing.instances[j];
+            instance_resources[inst.component_name + '_' + inst.task_id] = inst.instance_resources;
+          }
```

Comment:
can probably be replaced with a `map`

Reply:
Maybe next time. :p 

Map function is not available for dict. For loop might be more straightforward here.